plugins/entando-plugin-jpwebdynamicform/src/main/webapp/WEB-INF/plugins/jpwebdynamicform/apsadmin/jsp/message/messageFinding.jsp

```diff
@@ -3,295 +3,381 @@
-                <s:set var="group" value="#entityGroup" />
-
-                <div class="pager">
-                    <s:include value="/WEB-INF/apsadmin/jsp/common/inc/pagerInfo.jsp" />
-                    <s:include value="/WEB-INF/apsadmin/jsp/common/inc/pager_formBlock.jsp" />
-                </div>
+            <s:form action="search">
+                <p class="noscreen">
+                    <wpsf:hidden name="entityTypeCode"/>
+                    <wpsf:hidden name="from"/>
+                    <wpsf:hidden name="to"/>
+                    <wpsf:hidden name="answered"/>
+                    <s:iterator var="attribute" value="#searcheableAttributes">
+                        <s:if test="#attribute.textAttribute">
+                            <s:set var="textInputFieldName"><s:property value="#attribute.name"/>_textFieldName</s:set>
+                            <wpsf:hidden name="%{#textInputFieldName}"
+                                         value="%{getSearchFormFieldValue(#textInputFieldName)}"/>
+                        </s:if>
+                        <s:elseif test="#attribute.type == 'Date'">
+                            <s:set var="dateStartInputFieldName"><s:property
+                                    value="#attribute.name"/>_dateStartFieldName</s:set>
+                            <s:set var="dateEndInputFieldName"><s:property
+                                    value="#attribute.name"/>_dateEndFieldName</s:set>
+                            <wpsf:hidden name="%{#dateStartInputFieldName}"
+                                         value="%{getSearchFormFieldValue(#dateStartInputFieldName)}"/>
+                            <wpsf:hidden name="%{#dateEndInputFieldName}"
+                                         value="%{getSearchFormFieldValue(#dateEndInputFieldName)}"/>
+                        </s:elseif>
+                        <s:elseif test="#attribute.type == 'Number'">
+                            <s:set var="numberStartInputFieldName"><s:property value="#attribute.name"/>_numberStartFieldName</s:set>
+                            <s:set var="numberEndInputFieldName"><s:property
+                                    value="#attribute.name"/>_numberEndFieldName</s:set>
+                            <wpsf:hidden name="%{#numberStartInputFieldName}"
+                                         value="%{getSearchFormFieldValue(#numberStartInputFieldName)}"/>
+                            <wpsf:hidden name="%{#numberEndInputFieldName}"
+                                         value="%{getSearchFormFieldValue(#numberEndInputFieldName)}"/>
+                        </s:elseif>
+                        <s:elseif test="#attribute.type == 'Boolean'">
+                            <%-- DA FARE --%>
+                        </s:elseif>
+                        <s:elseif test="#attribute.type == 'ThreeState'">
+                            <%-- DA FARE --%>
+                        </s:elseif>
+                    </s:iterator>

```

Comment:
Per quanto riguarda questi commenti, erano già presenti al momento della mia presa in carico della carta.